packages/ui-view/src/View/styles.ts

```diff
@@ -434,33 +440,33 @@ const generateStyle = (
       overscrollBehavior: overscrollBehavior ? overscrollBehavior : 'auto',
       ...(withVisualDebug
         ? {
-            boxShadow: `0 0 0 1px ${componentTheme.debugOutlineColor}`
-          }
+          boxShadow: `0 0 0 1px ${componentTheme.debugOutlineColor}`
+        }
         : {}),
       ...(withBorder(props)
         ? {
-            borderStyle: componentTheme.borderStyle,
-            ...(borderColorVariants[borderColor!] || {
-              borderColor: borderColor
-            })
-          }
+          borderStyle: 'solid',
```

Comment:
I dont see this in the token changelog

Reply:
It has been already added to the v12, when @balzss texted me